cron/config/config.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Vershandlern 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permisshandlerns and
+// limitathandlerns under the License.
+
+package config
+
+const (
+	ResultDataBucketURL string = "gs://ossf-scorecard-data"
+	RequestTopicURL            = "gcppubsub://projects/openssf/topics/scorecard-batch"
```

Comment:
Ideally, project name and gcs bucket should go in a config.yaml and read from there. If some env var is set (e.g. someone's dockerfile who wants to run scorecard service on their project (although rare)), it can use it as an override, otherwise use the default from config.yaml. for tests, you can just set those env vars. this should avoid dependency injection. wdyt @naveensrinivasan @azeemshaikh38 @oliverchang

Reply:
I agree with that https://12factor.net/config. Thanks

Reply:
I have added a TODO for this. Will have a follow up PR addressing this to avoid adding more logic in this PR.